src/assets/css/main.css

```diff
@@ -213,6 +213,26 @@ hr {
   overflow-wrap: anywhere;
 }
 
+.avatar-name-container {
+  display: flex !important;
+  flex-direction: row;
+  align-items: center;
+  line-height: 1.2rem;
+}
+
+.list-flex-v .avatar-name-container {
+    margin-top: 0.5rem;
+}
+
+.list-flex-h .avatar-name-container {
+    margin-left: 0.5rem;
+    margin-right: 0.5rem;
+}
+
```

Comment:
These custom flex classes shouldn't be necessary: use bootstraps flex utilities: https://getbootstrap.com/docs/4.0/utilities/flex/